server/src/test/java/com/objectcomputing/checkins/services/role/role_permissions/RolePermissionsControllerTest.java

```diff
@@ -85,33 +84,33 @@ void testAssignPermissionToRole() {
         MemberProfile sender = createADefaultMemberProfile();
         assignAdminRole(sender);
 
-        Role memberRole = getRoleRepository().findByRole(RoleType.MEMBER.name()).orElseThrow();
-        Permission birthdayPermission = getPermissionRepository().findByPermission(Permissions.CAN_VIEW_BIRTHDAY_REPORT.name()).get(0);
+        Role memberRole = getRoleRepository().findByRole(RoleType.MEMBER.name()).get();
```

Comment:
I noticed that the `testGetAllRolePermissions` method still uses `.orElseThrow()` while `testAssignPermissionToRole` and `testRemovePermissionFromRole` were changed to use `.get()`. To me it makes sense to use `orElseThrow()` since these tests should fail if they can't get past that step.